.github/workflows/api-breaking-changes.yml

```
@@ -0,0 +1,96 @@
+        with:
+          ref: main
+          path: main
+
+      - name: Setup Elixir
+        uses: erlef/setup-beam@v1
+        with:
+          elixir-version: '1.15.4'
+          otp-version: '26.0'
+
+      - name: Cache dependencies
+        uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)